docker-compose.test.yml

```diff
@@ -33,6 +33,8 @@ services:
     image: postgres:12.4
     container_name: test-db
     env_file: .env
+    ports:
+      - "5432:5432"
     networks:
```

Comment:
Does the db need to expose 5432 in docker-compose.test.yml? We aren't interacting with the db during tests are we?  This would prevent running tests while you have a local instance running (something I frequently do). I am unclear on what benefit this brings.

Reply:
We need to expose a port to pass the (new) `-l` flag to `run-tests` but check out the change I just made to that. It addresses this problem.